### ISSUE TITLE
pgweb hangs when no internet is available.

### ISSUE THREAD

**Issue by tlorens** [sosedoff/pgweb#511]
Working on a plane without internet, I noticed pgweb took 12 seconds for it to show table contents or any action for that matter. Strangely odd when everything is localhost anyhow. 

I'm no Go expert, so I don't think I'm capable of tracking this down or even understanding why this is happening.

**Comment by sosedoff** [comment]
You'll need to provide a bit more information for me to understand the issue:

1) do you have the request logs from the terminal? it shows duration for every request
2) how did you measure that 12 seconds? was it from the logs or a rough visual estimate?
3) which browser did you use?

**Comment by tlorens** [comment]
Using a Google Pixelbook (Version 89.0.4389.95 (Official Build) (64-bit)).  I only knew about the delay by looking at the pgweb console terminal logs.  I'm unable to recreate the issue now.  I've turned off WiFi and pgweb seems to work as expected.  Not entirely sure what would cause those delays each and every click in the UI.  

Closing this for now. I'll reopen if I'm able to provide more information if it happens again.
